src/tree/private_tree.rs

```diff
@@ -134,13 +134,13 @@ impl PrivateTree {
     ) {
         let hash_len = ciphersuite.hash_length();
         let start_secret = match start_secret {
-            Some(secret) => hkdf_expand_label(ciphersuite, secret, "path", &[], hash_len),
+            Some(secret) => secret.to_vec(),
```

Comment:
This appears to be wrong. According to spec we have `path_secret[0] = DeriveSecret(leaf_secret, "path")`. So unless you changed the way this function is called the `start_secret` shouldn't be used directly.

Reply:
You are right and that's described here: https://github.com/raphaelrobert/maelstrom/issues/165
It's already fixed in the follow-up PR.